football/lib/flagMapper.js

```diff
@@ -0,0 +1,72 @@
+const flagForTeam = (teamName) => {
```

Comment:
This file isn't used any more but are we keeping it for the future?

Reply:
yes, once we find a way to use emojis in the runtime.js